Firebase/Auth/Source/Public/FIRFederatedAuthProvider.h

```diff
@@ -24,6 +24,9 @@
 
 NS_ASSUME_NONNULL_BEGIN
 
+/**
+    Utility class for constructing federated auth provider credentials.
```

Comment:
"class" seems incorrect here since it's a protocol?

Reply:
Fixed.